helper.php

```diff
@@ -425,6 +425,22 @@ public function getAllTags($namespace = '', $order_by = 'tag', $desc = false) {
         return $db->res2arr($res);
     }
 
+    /**
+     * Get all pages with tags and their tags
+     *
+     * @return array ['pid' => 'tag1,tag2,tag3']
```

Comment:
As a helper function, it would probably be cleaner if the tags were in an array instead of being comma separated.